oterm/app/widgets/chat.py

```diff
@@ -90,6 +90,8 @@ async def load_messages(self) -> None:
             chat_item.text = message
             chat_item.author = author
             await message_container.mount(chat_item)
+        autoscroll_static = Static("", classes="autoscroll", disabled=True)
```

Comment:
Is this necessary? Can we reuse the "loading" indicator instead?
If it is necessary, it should be renamed to something more meaningful like 'container-end'.